conference/dataaccess.py

```diff
@@ -595,12 +596,15 @@ def profile_data(uid, preload=None):
     except KeyError:
         bio = profile.getBio()
 
-    talks_map = {
-        'by_conf': {'all': []},
-        'accepted': {'all': []},
-        'proposed': {'all': []},
-        'canceled': {'all': []},
-    }
+    # NOTE(artcz)(2018-05-31)
+    # I updated it to automatically fill the map with all available statuses.
+    # (previously it was staticly defined dict).
+    # However... I don't know why it's there, maybe we would be better with
+    # defaultdict?
```

Comment:
Not sure either, but this talk data is passed around in the code a lot. Perhaps this is used for implementing the user talk schedule or simply as a way to cache the talks data by category.